### DISCUSSION TITLE
ProgressIndicator Material3

### DISCUSSION THREAD

**Discussion by neronguyen** [Tlaster/PreCompose#260] [Q&A] [answered]
java.lang.NoSuchMethodError: No virtual method
I have met that exception using material3.
I already implement some dependencies.
```
     commonMain.dependencies {
            api(compose.runtime)
            api(compose.material3)
            api(compose.foundation)
            api(compose.animation)
```
Thanks for helping...

**Comment by Tlaster** [comment, marked as answer]
I suggest you'd better to open a discussion in [compose-multiplatform](https://github.com/JetBrains/compose-multiplatform) since it has nothing to do with PreCompose.

**Reply by neronguyen** [reply]
Thankss
